src/OXRS_Input.cpp

```diff
@@ -102,6 +102,44 @@ void OXRS_Input::process(uint8_t id, uint16_t value)
   }
 }  
 
+void OXRS_Input::query(uint8_t id) 
+{
+  // Check if we have a callback to handle the events
+  if (_callback) 
+  {
+    // Read security sensor values in quads (a full port)
+    uint8_t securityCount = 0;
+
+    for (uint8_t i = 0; i < INPUT_COUNT; i++)
+    {
+      // Get the type and current state of this input
+      uint8_t type = getType(i);
+      uint8_t state = _state[i].data.state;
+
+      // Only makes sense to publish the current state for CONTACT, SECURITY and SWITCH inputs
+      switch (type) 
+      {
+        case CONTACT:
+        case SWITCH:
+          // Ignore if we are in the middle of debounce checking
+          if (state == IS_HIGH)
+            _callback(id, i, type, HIGH_EVENT);
+          else if (state == IS_LOW)
+            _callback(id, i, type, HIGH_EVENT);          

```

Comment:
```C
 _callback(id, i, type, HIGH_EVENT);  
```
shouldn't be this `LOW_EVENT` ?

Reply:
Yes! Good catch, thanks for having a look.